### PR TITLE
completed

### PR DESCRIPTION
**Pull request by bug-smith**
Completed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,17 @@
   "scripts": {
     "prepare": "husky install"
   },
+  "nodemonConfig": {
```
Comment (RobertGardner) [inline package.json]: This one will only go in the exercise package.json, not the one at the root.

File: package.json [hunk 2]
```diff
@@ -6,7 +6,17 @@
...
+    "execMap": {
+      "ts": "ts-node-esm",
+      "js": "ts-node-esm"
+    }
+  },
   "devDependencies": {
+    "@types/express": "^4.17.20",
```
Comment (RobertGardner) [inline package.json]: @types/express and @types/node also only in the exercise where they're used. The -eslint ones should be here at the root, though.

**Review by RobertGardner — CHANGES_REQUESTED** [review]
Minor fixes, still.

File: package.json (hunk 2 above)
Comment (bug-smith) [reply package.json]: got rid of them from the root, will update per excerise going forward.

**Pull request merged**